### ISSUE TITLE
Use oAuth2 protocol for the API

### ISSUE THREAD

**Issue by vpmalley** [vpmalley/jetlag#56]
Il y a Passeport pour ça, mais il faudra une nouvelle version de laravel pour ça
https://laravel.com/docs/5.3/passport

**Comment by Vallenain** [comment]
Do we need it at the moment ? I think the cookie-based session prevents non authorized users to access our API endpoints, right ?
(about the Laravel upgrade, I'm ok with it, cf https://github.com/vpmalley/jetlag/issues/19#issuecomment-262387453 :) )

**Comment by vpmalley** [comment]
yeah it's a long-term feature (not in the beta milestone). I created this isssue to offload it from my memory
